runtime/onert/core/src/exec/Executors.cc

```diff
@@ -87,21 +87,58 @@ void Executors::execute(const IODescription &desc)
 
 void Executors::checkSupportedMultimodel() const
 {
-  // Assume 2 executors only
-  // Assume that each model may have only one subgraph
-  // TODO Support general case
-  if (_executors.size() != 2)
-    throw std::runtime_error{"NYI: Multi model execution for this package is not supported yet"};
+  // Assumption
+  // Models
+  //   size: n
+  //   sizeof(model(a).outputs) == sizeof(model(a+1).inputs)
+  auto const model_count = modelCount();
```

Comment:
(Just note) I wondered why `model_count` is required when iterating with `for :`. I've found `model_count` is used at the last for-loop.